systemrequirements.adoc

```diff
@@ -266,13 +290,29 @@ including:
 Supported devices for the {product-short} Touch Client include:
 
 * iOS6+: iPad®, iPad mini®, iPhone®, iPod touch®
-* Android 4.0+: Nexus 7, Nexus 10, Samsung Galaxy Tab,
-  Samsung Galaxy S® III, Samsung Galaxy S® 4, Galaxy Nexus
+* Android 4.0+: Nexus 7, Nexus 10, Samsung Galaxy Tab™,
+  Samsung Galaxy S® III, Samsung Galaxy S® 4, Galaxy Nexus™
+endif::networkeditiondoc[]
+
+ifdef::networkeditiondoc[]
+[[BES]]
```

Comment:
I thought BES is dead (our last release was for 7.2.x, no?).  Seems like this should be removed.